src/main/java/com/nnaltakyan/core/communications/domain/sentemail/model/SentEmail.java

```diff
@@ -52,4 +52,8 @@ public class SentEmail extends BaseEntity
 	@NotNull
 	@Column(nullable = true)
 	private Instant sent;
+
+	@Column
+	@Enumerated(EnumType.STRING)
+	private EmailSentStatus status = EmailSentStatus.PENDING;
```

Comment:
please change this part, add the default value as a @Column's argument

Reply:
Done.